components/nimbus/src/lib.rs

```diff
@@ -210,25 +238,109 @@ impl NimbusClient {
 
     pub fn apply_pending_experiments(&self) -> Result<Vec<EnrollmentChangeEvent>> {
         log::info!("updating experiment list");
+        // If the application did not pass in an installation date,
+        // we check if we already persisted one on a previous run:
         let db = self.db()?;
         let mut writer = db.write()?;
+
+        let installation_date = self.get_installation_date(db, &mut writer)?;
+        let update_date = self.get_update_date(db, &mut writer)?;
+        let now = Utc::now();
+        let days_since_install = now - installation_date;
+        let days_since_update = now - update_date;
```

Comment:
Not sure of the API, but does `Utc::now() - DateTime<Utc>` gives a duration back in `days`?

Reply:
The name was a bad name 😄 , changed the name to `duration_since_install` which includes the day information I later extract